log/sockstatlog/logger.go

```diff
@@ -137,19 +155,19 @@ func (l *Logger) Write(p []byte) (int, error) {
 }
 
 // poll fetches the current socket stats at the configured time interval,
-// calculates the delta since the last poll, and logs any non-zero values.
+// calculates the delta since the last poll, and writes any non-zero values to the logger event channel.
 // This method does not return.
 func (l *Logger) poll() {
 	// last is the last set of socket stats we saw.
 	var lastStats *sockstats.SockStats
 	var lastTime time.Time
 
-	enc := json.NewEncoder(l)
+	ticker := time.NewTicker(pollInterval)
 	for {
 		select {
 		case <-l.ctx.Done():
 			return
```

Comment:
Should we call `ticker.Stop()` here (since we don't have a `Stop()` call in `Shutdown()` anymore)? The docs for `NewTicker` says "Stop the ticker to release associated resources."

Reply:
yeah, good call.  Added to both poll and logEvent, since they each maintain their own tickers.